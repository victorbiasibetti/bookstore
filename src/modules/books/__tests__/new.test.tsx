import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import New from "../new";

configure({ adapter: new Adapter() });

describe("Test New Book render", () => {
  it("Render", () => {
    const wrapper = shallow(<New />);
    expect(wrapper).toMatchSnapshot();
  });
});
