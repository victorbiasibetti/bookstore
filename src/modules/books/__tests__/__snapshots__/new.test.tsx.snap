// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test New Book render Render 1`] = `ShallowWrapper {}`;
